lib/bango/errors.py

```diff
@@ -53,3 +56,13 @@ class ProcessError(Exception):
 
     def __init__(self, response):
         self.response = response
+
+
+class StatusError(ParseError):
+
+    """
+    This is an error that occurs in the Stats checking code, which we do want
+    to commit so we've got a history of the transaction failure.
+    """
+
+    _do_not_rollback = True
```

Comment:
That's a pretty neat idea. I don't think it applies to this view, but its probably better to set it up correctly the first time. I'll noodle on this for a little bit.